azurerm/internal/services/network/tests/virtual_hub_connection_resource_test.go

```diff
@@ -257,6 +293,22 @@ func testCheckAzureRMVirtualHubConnectionExists(resourceName string) resource.Te
 	}
 }
 
+func testCheckAzureRMVirtualHubConnectionDoesNotExist(resourceGroupName, vhubName, vhubConnectionName string) resource.TestCheckFunc {
```

Comment:
We will need to use the new data.CheckWithClient here, otherwise this will fail when we move to binaryb testing

Reply:
Updated. Thx.